data/json/items/melee/bludgeons.json

```
@@ -1485,5 +1485,24 @@
     "to_hit": { "grip": "weapon", "length": "long", "surface": "line", "balance": "good" },
     "price": 16000,
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "millwall_brick",
+    "name": { "str": "millwall brick" },
```

```suggestion
    "name": { "str": "Millwall brick" },
```
Millwall refers to Millwall Football Club, so it could be capitalized.